consensus/processor.go

```diff
@@ -8,12 +8,23 @@ import (
 	"github.com/filecoin-project/go-filecoin/actor"
 	"github.com/filecoin-project/go-filecoin/actor/builtin/account"
 	"github.com/filecoin-project/go-filecoin/address"
+	"github.com/filecoin-project/go-filecoin/metrics"
 	"github.com/filecoin-project/go-filecoin/state"
 	"github.com/filecoin-project/go-filecoin/types"
 	"github.com/filecoin-project/go-filecoin/vm"
 	"github.com/filecoin-project/go-filecoin/vm/errors"
 )
 
+var consensusRegistry *metrics.Registry
+var pbTimer *metrics.Float64Timer
+var amTimer *metrics.Float64Timer
+
+func init() {
+	consensusRegistry = metrics.NewRegistry()
```

Comment:
I'm not sure where a good place for this is yet, I don't think it belongs here (dare I say we put it on node?). However since it is only referenced here I think we can leave it for a follow on.